### PR TITLE
LPS-202692 NoSuchLayoutException occurs when going to staging configuration via Portlet

### PR DESCRIPTION
**Pull request by DaniSzimko**
https://liferay.atlassian.net/browse/LPS-202692

### CODE REVIEW

**Comment by DaniSzimko** [comment]
ci:test:relevant

**Comment by DaniSzimko** [comment]
ci:test:staging

**Comment by DaniSzimko** [comment]
ci:test:sf

**Comment by DaniSzimko** [comment]
ci:test:staging

File: modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingImplTest.java [hunk 1]
```diff
@@ -215,6 +217,51 @@ public void testGetRemoteLayout() throws Exception {
...
+				_remoteLiveGroup.getGroupId(),
+				remoteStagingGroupLayout.isPrivateLayout(),
+				remoteStagingGroupLayout.getLayoutId());
+
+			Assert.assertNotNull(remoteLiveGroupLayout);
+
+			Assert.assertEquals(
+				remoteLiveGroupLayout.getPlid(), remoteLiveGroupLayoutPlid);
+		}
+		catch (NoSuchLayoutException noSuchLayoutException) {
```
Comment (4lejandrito) [inline modules/apps/staging/staging-test/src/testIntegration/java/com/liferay/staging/test/StagingImplTest.java:260]: hey @DaniSzimko I think this is redundant. We can remove the `try` block and just let the exception bubble up, which will cause the test to fail anyway, and with a much clearer error message.

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

**Comment by DaniSzimko** [comment]
continue on : https://github.com/liferay-headless/liferay-portal/pull/1748

cc @vendeltoreki , @4lejandrito

**Pull request closed without merging**